test/files/blt001/fail-blt001-scenario02-door_type_own_operation_type.ifc

```diff
@@ -0,0 +1,32 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2024-06-14T16:36:44',(''),(''),'IfcOpenShell-v0.7.0-f7c03db75','IfcOpenShell-v0.7.0-f7c03db75','');
+FILE_SCHEMA(('IFC4X3_ADD2'));
+ENDSEC;
+DATA;
+#1=IFCPERSON($,$,'',$,$,$,$,$);
+#2=IFCORGANIZATION($,'',$,$,$);
+#3=IFCPERSONANDORGANIZATION(#1,#2,$);
+#4=IFCAPPLICATION(#2,'v0.7.0-f7c03db75','IfcOpenShell-v0.7.0-f7c03db75','');
+#5=IFCOWNERHISTORY(#3,#4,$,.NOTDEFINED.,$,#3,#4,1718383004);
+#6=IFCDIRECTION((1.,0.,0.));
+#7=IFCDIRECTION((0.,0.,1.));
+#8=IFCCARTESIANPOINT((0.,0.,0.));
+#9=IFCAXIS2PLACEMENT3D(#8,#7,#6);
+#10=IFCDIRECTION((0.,1.));
+#11=IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,1.E-05,#9,#10);
+#12=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#13=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#14=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17=IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453292519943295),#16);
+#18=IFCCONVERSIONBASEDUNIT(#12,.PLANEANGLEUNIT.,'DEGREE',#17);
+#19=IFCUNITASSIGNMENT((#13,#14,#15,#18));
+#20=IFCPROJECT('2meTAdIsTAwOKSutMNpxgA',#5,'',$,$,$,$,(#11),#19);
+#21=IFCDOORTYPE('0AvhKGa3DC4O3_85l41YQd',$,$,$,$,$,$,$,$,$,$,$,$);
```

Comment:
```suggestion
#21=IFCDOORTYPE('0AvhKGa3DC4O3_85l41YQd',$,$,$,$,$,$,$,$,$,.SLIDING_TO_RIGHT.,$,$);
```

Technically this is correct as a fail since there is `IfcRelDefinesByType` but let's add a type to `IfcDoorType` to make it "fully" wrong.